scavenger-frontend/src/components/snapshot/SnapshotTable.vue

```diff
@@ -165,23 +166,31 @@ export default {
           ElNotification.success({message: this.$t("message.snapshot.refresh-success")});
         })
     },
-    exportSnapshot(id) {
-      const fileName = `snapshot${id}.tsv`;
-      this.$http.get(`/customers/${this.customerId}/snapshot/${id}/export?fn=${fileName}`)
-        .then((response) => {
-          const csvFile = new Blob([response.data], {type: 'text/csv'});
-          const downloadLink = document.createElement("a");
-          downloadLink.download = fileName;
-          downloadLink.href = window.URL.createObjectURL(csvFile);
-          downloadLink.style.display = "none";
-          document.body.appendChild(downloadLink);
-          downloadLink.click();
-          window.URL.revokeObjectURL(downloadLink.href);
-          document.body.removeChild(downloadLink);
-        })
-        .catch(() => {
-          ElNotification.error({message: this.$t("message.snapshot.export-fail")});
-        });
+    async exportSnapshot(id) {
+      const header = ["filterInvokedAtMillis", "packages", "status", "excludeAbstract", "parent", "signature", "type", "usedCount", "unusedCount", "lastInvokedAtMillis"];
+      const workBook = utils.book_new();
+      const workSheet = utils.json_to_sheet([], {header});
+      const loader = this.$loading.show()
+      try {
+        const page = await this.$http.get(`/customers/${this.customerId}/snapshots/${id}/export/total-page`).then(res => res.data.totalPage);
+
+        for (let idx = 0; idx <= page; idx++) {
+          const response = await this.$http.get(`/customers/${this.customerId}/snapshots/${id}/export?page=${idx}`)
+          const data = response.data.map(row => {
+            row.filterInvokedAtMillis = Moment.unix(row.filterInvokedAtMillis).format("YYYY.MM.DD HH:mm:ss");
+            row.lastInvokedAtMillis = row.lastInvokedAtMillis == null ? null : Moment.unix(row.lastInvokedAtMillis).format("YYYY.MM.DD HH:mm:ss");
+            return row;
+          });
+          utils.sheet_add_json(workSheet, data, {header, origin: -1, skipHeader: true})
+        }
+
+        utils.book_append_sheet(workBook, workSheet);
+        writeFile(workBook, `snapshot_${id}.csv`);
```

Comment:
@sohyun-ku 

Could u explain why xlsx should be created in fe layer? what was your intention here?

Reply:
I opened this issue #69.
In my opinion... it has the advantage of not handling IO on the server and doesn't require direct control of FE's DOM.

Reply:
I don't think this is really necessary.
This fix blocks the way directly download excel using curl. Moreover, it always need to use the browser.
Instead of this... what about adding excel download beside of csv?

Reply:
I hadn't considered using curl too..haha..

> what about adding excel download beside of csv?

Before I did this PR, there was already excel download feature, what does this mean?

Reply:
@junoyoon Please check out sohyun`s comment.

Reply:
@sohyun-ku @taeyeon-Kim 
Sorry for late response. I was too busy.. :-(

It was csv export feature not excel.
And csv is created in server side.
So it's callable using cli

I think we should make it server side job (not fe).

Reply:
@junoyoon @taeyeon-Kim 

thanks for explaining :)
I didn't think someone can use cli when using csv export feature...
We have already a csv export feature on the server side before this PR.
So I'll close this PR.